Echo/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ set( PSYNC_HOME
     )
 
```

Comment:
It doesn't look like all the examples have install rules but I like the idea of having them. Adding the following to this file should do the trick but probably test it just to make sure.


install(
   TARGETS polysync-echo
   DESTINATION ${PSYNC_HOME}/bin
   )